.gitignore

```diff
@@ -8,19 +8,11 @@
 # Java class files
 *.class
 
-# generated files
-bin/
-build/
-*.iml
-gen/
-target/
-
```

Comment:
Why do you remove this generated files from gitignore?

Reply:
bin, gen and target is Eclipse
build in in L38 too
.iml in in L37 too